### ISSUE TITLE
`check-test-warnings` workflow is failing

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/effectsize#565]
New warning in the test suite: https://github.com/easystats/effectsize/actions/runs/4251413352/jobs/7393734891

```r
── Error (test-eta_squared.R:561): car MVM ─────────────────────────────────────
Error in `summary.Anova.mlm(model)`: (converted from warning) HF eps > 1 treated as 1
Backtrace:
    ▆
 1. └─effectsize::eta_squared(A_car, ci = NULL) at test-eta_squared.R:561:2
 2.   ├─effectsize:::.anova_es(...)
 3.   └─effectsize:::.anova_es.Anova.mlm(...)
 4.     ├─base::summary(model)
 5.     └─car:::summary.Anova.mlm(model)
```

**Comment by mattansb** [comment]
Hmmm I can't reproduce this on my machine... Can you?

**Comment by mattansb** [comment]
Ah, it seems Daniel has already fixed this in #563
